app/assets/javascripts/modules/gtm-reorder-listener.js

```diff
@@ -0,0 +1,19 @@
+var GtmReorderListener = {}
+
+GtmReorderListener.handleReorder = function (eventName) {
+  return function (event) {
+    var element = event.target
+
+    if (!element) {
+      return
+    }
+
+    window.dataLayer.push({
+      event: eventName
```

Comment:
We seem to be missing the `dataGtm` attribute now 😬. I think we should follow [the approach for topics](https://github.com/alphagov/content-publisher/blob/master/app/assets/javascripts/modules/gtm-topics-listener.js#L3), here. We could have two handlers: `handleClick` and `handleDrag`.

Happy to chat some more if that helps!

Reply:
I think I get what you mean. It felt a bit unnecessary to me to have both `eventName` and `dataGtm` passed to the listener as we can deduce them, but have updated now.